partner_restrict_payment_acquirer/models/payment_acquirer.py

```diff
@@ -0,0 +1,26 @@
+from odoo import api, models
+
+
+class PaymentAcquirer(models.Model):
+    _inherit = "payment.acquirer"
+
+    @api.model
+    def get_allowed_acquirer(self, acquirers, invoice_id=None, order_id=None):
+        """
+        Get allowed acquirers by the customer
+        :param list acquirers: list of acquirers
```

Comment:
Why do we need to pass the list of acquires as an agument?
Wouldn’t it be more reasonable to compose it  it inside this function?

Reply:
Resolved